Source/VideoView.swift

```diff
@@ -121,8 +121,8 @@ class VideoView: UIView {
         }
     }
 
-    func prepare(using viewerItem: ViewerItem, completion: @escaping (Void) -> Void) {
-        self.addPlayer(using: viewerItem) {
+    func prepare(using viewable: Viewable, completion: @escaping (Void) -> Void) {
```

Comment:
I think the protocol should be called `Viewable` but the instance conforming to that protocol should be a noun right? Isnt it viewableController?